tests/unit/BusinessStartDate.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('Business Start Date', () => {
   let wrapper: any
 
   beforeEach(async () => {
-    store.stateModel.businessInformation.foundingDate = '2021-07-01T00:00:00.000000+00:00'
+    store.stateModel.businessInformation.foundingDate = '2021-07-01T12:00:00.000000+00:00'
```

Comment:
Same as other PR -- I'm using 12:00 since it's not affected by standard/daylight changes.